person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  # Attribute accessor setters and getters
+  attr_accessor :id, :name, :age
+
```

Comment:
- [ ] As required  for the project, kindly ensure that the `:id` is only able to be retrieved so only the `Getter` should be define to ir i.e. `attr_reader`.